### ISSUE TITLE
There seems to be no way to delete a "branch" using the GUI editor

### ISSUE THREAD

**Issue by power833** [RoamJS/query-builder#232]
There seems to be no way to delete a "branch" using the GUI editor. Am I correct? This prevents queries from working, as sometimes I am left with empty branches that prevent the query from working properly and I can't delete them from the GUI.

Thanks

**Comment by dvargas92495** [comment]
could you share a video clarifiying the issue?

**Comment by mdroidian** [comment]
@power833 you are correct, there is no UI button to delete created branches, unfortunately.

This will be added to the backlog as a bug 😀.

In the meantime, to delete branches you can go to the blocks below they query.
Go to `scratch`, then `conditions`, and there you will find the `or` branches.  

After you delete the branches you don't want, refresh the `{{query block}}` by putting your cursor over the text and removing it.

Here is a video demonstrating:
https://www.loom.com/share/df7f2340170c45f5852cdc965df5b913
